src/tests/ftest/daos_test/daos_core_test.yaml

```diff
@@ -58,15 +58,21 @@ daos_tests:
   num_clients:
     num_clients: 1
   Tests: !mux
-    test_d:
-      daos_test: d
-      test_name: DAOS degraded-mode tests
-      test_timeout: 450
-      scalable_endpoint: True
-    test_m:
-      daos_test: m
-      test_name: Management tests
-      test_timeout: 110
+# Skip for ticket DAOS-6311
+# Temporary using comments to skip
+# WIP to refactor so subtest can be skipped
+# using skipForTicket
+#    test_d:
+#      daos_test: d
+#      test_name: DAOS degraded-mode tests
+#      test_timeout: 450
+#      scalable_endpoint: True
+#
+# Skip for ticket DAOS-6205
+#    test_m:
+#      daos_test: m
+#      test_name: Management tests
+#      test_timeout: 110
```

Comment:
I would much rather see this implemented by adding the following in src/tests/ftest/daos_test/daos_core_test.py:

```
    def setUp(self):
        """Set up each test case."""
        test_name= self.params.get("test_name")
        if test_name == "DAOS degraded-mode tests":
            self.cancelForTicket("DAOS-6311")
        if test_name == "Management tests":
            self.cancelForTicket("DAOS-6205")
```

Without any changes to src/tests/ftest/daos_test/daos_core_test.yaml

Reply:
Done. Thanks.